pkg/lib/common.js

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Hmm, we now have utils.tsx and common.js . utils' `is_json_dict` would fit quite well in here, the manifest and fmt_to_fragments should indeed stay separate.

Reply:
Sure.  I'll move it over.

Reply:
Let's keep it for now. This got superseded by the later discussion about moving to `_internal`.